### DISCUSSION TITLE
Custom style and custom JS doesn't appears

### DISCUSSION THREAD

**Discussion by Bajweb** [coreybutler/nvm-windows#728] [Support]
Hello
I am using latest understarp theme and node version. Changes that I provide on /src/sass/theme/_theme.scss file doesn't appears on website. Terminal is running ok, no errors, even the code is compile and minify - I can see it in /css/theme.min.css file. Also the changes in php files like header.php or footer.php are applying too, but style( scss ) and JS doesn't. Does anyone know now what the problem may be?

Terminaal 
> understrap@1.1.0 css
> npm-run-all css-compile css-postcss css-minify

> understrap@1.1.0 css-compile
> sass --style expanded --source-map --embed-sources --no-error-css --quiet src/sass/theme.scss:css/theme.css src/sass/custom-editor-style.scss:css/custom-editor-style.css

> understrap@1.1.0 css-postcss
> postcss --config src/build/postcss.config.js --replace "css/*.css" "!css/*.rtl*.css" "!css/*.min.css"

> understrap@1.1.0 css-minify
> cleancss -O1 --format breakWith=lf --with-rebase --source-map --source-map-inline-sources --output css/ --batch --batch-suffix ".min" "css/*.css" "!css/*.min.css" "!css/*rtl*.css"

[nodemon] clean exit - waiting for changes before restart
[Browsersync] File event [change] : css\custom-editor-style-bootstrap4.min.css
[Browsersync] File event [change] : css\custom-editor-style.min.css
[Browsersync] File event [change] : css\theme-bootstrap4.min.css
[Browsersync] File event [change] : css\theme.min.css

Thanks Alex

**Comment by coreybutler** [comment]
This doesn't have anything to do with NVM for Windows. You will have better luck getting your question answered at the [understrap repo](https://github.com/understrap/understrap).
